packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -49,7 +49,28 @@ function generateItems(items) {
   return html.join('');
 }
 
-
+function generateSelectableItems(items, context){
+
+  const html = items.map(({ content, color, selected }, index) => {
+    return `<cbp-structured-list-item ${color != 'default' ? `color="${color}"` : ''} ${selected ? `selected` : ''} >
+    <cbp-checkbox 
+      value= ${index}
+      ${context && context != 'light-inverts' ? `context=${context}` : ''}   
+    >
+      <label>
```

Comment:
the checkbox component renders the `label` tag, so you shouldn't add it here. You just needed to add the text.

Reply:
removed label tag here